tmail-backend/webadmin/webadmin-mailbox/src/test/java/com/linagora/tmail/webadmin/cleanup/CleanupTrashServiceTest.java

```diff
@@ -44,31 +47,33 @@ public class CleanupTrashServiceTest {
     private UsersRepository usersRepository;
     private JmapSettingsRepository jmapSettingsRepository;
     private StoreMailboxManager storeMailboxManager;
+    private SystemMailboxesProvider systemMailboxesProvider;
     private Clock clock;
 
     @BeforeEach
     void setUp() {
         InMemoryIntegrationResources resources = InMemoryIntegrationResources.defaultResources();
         storeMailboxManager = resources.getMailboxManager();
+        systemMailboxesProvider = new SystemMailboxesProviderImpl(storeMailboxManager);
         usersRepository = mock(UsersRepository.class);
```

Comment:
usersRepository = MemoryUsersRepository.withVirtualHosting(domainList);
        usersRepository.addUser(BOB, "anyPassword");

Reply:
Generally, avoid using mock whenever possible is better.